xcube/constants.py

```diff
@@ -64,6 +65,9 @@
     AUTHORITY["EPSG","4326"]]
 """
 
+CRS84 = 'OGC:CRS84'
+CRS_CRS84 = pyproj.crs.CRS.from_string('OGC:CRS84')
```

Comment:
```suggestion
CRS_CRS84 = pyproj.crs.CRS.from_string(CRS84)
```

I'm being pedantic, but this is a slight improvement :)

Reply:
Ya but make sense, can do it :)